scout/data-handling.md

```diff
@@ -0,0 +1,57 @@
+---
+description: How Docker Scout handles image metadata
+keywords: scanning, supply chain, security, data, metadata
+title: Data collection and storage in Docker Scout
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout image analysis works by collecting metadata from the container
+images that you analyze. That metadata is stored on the Docker Scout platform.
+
+## Data transmission
+
+Docker Scout collects and sends the following image metadata to the platform:
+
+**Docker and OCI image metadata**:
```

Comment:
Is this an actual thing in the UI or a stylistic choice?

Reply:
it was just a (poor) stylistic choice, updated :)